util/envDynawo.sh

```diff
@@ -1574,26 +1647,84 @@ create_distrib_with_omc() {
   # Deploy Dynawo
   deploy_dynawo
 
+  if [ "$DYNAWO_COMPILER" = "GCC" ]; then
+    gcc_version=$($DYNAWO_C_COMPILER -dumpversion)
+    if [ $(echo $gcc_version | cut -d '.' -f 1) -lt 5 ]; then
+      sed -i 's/FLAGS="$FLAGS -std=c++98"/FLAGS="$FLAGS -std=c++98 -D_GLIBCXX_USE_CXX11_ABI=0"/' $DYNAWO_DEPLOY_DIR/sbin/compileCppModelicaModelInDynamicLib
```

Comment:
Why is this test done here instead of inside the compileCppModelicaModelInDynamicLib script?

Reply:
This means that we have to write code compatible with old ABI, is this really required? Maybe we could create the distribution on a slightly more recent linux distribution?

Reply:
It is done here because no reason to do it inside compileCppModelicaModelInDynamicLib if you don't distribute the binaries. The problem will appear if you distribute a binary compiled with gcc4 and try to compile models with a newer gcc7 for example, the string ABI will differ and symbols would not be found. Wde are already compatible with an old ABI as we compile with gcc4 but by default newer compiler will use the new one and std=c++98 is not enough to be striclty compatible.
https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html

Reply:
Why not create a distribution with a newer version of gcc?

Reply:
Also it could be done inside the script with an option. Doing an sed is dependent on the way the script is written. If we modify the script, we might break this silently.

Reply:
Best way to make the binaries work on most platforms is to use an old gcc if we start using gcc5.1 as the base one the binaries will not be compatible on centos7...
The other way of doing the test would be in CMake when we install the file and check which version of the compiler is used to enforce the ABI
A test on the binaries produced could be done on dynawo-nightly to run a simulation with compilation on the fly and we would catch the problem at each new release but not daily...

Reply:
OK for the test